notification/docs/IntegrationGuide.md

```diff
@@ -0,0 +1,67 @@
+# Integration of payment into checkout process
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Contents**
+
+  - [Glossary](#glossary)
+  - [Requirements](#requirements)
+      - [for CTP project:](#for-ctp-project)
+      - [for the Node.js:](#for-the-nodejs)
+  - [Environment variables to configure the module:](#environment-variables-to-configure-the-module)
+- [After deployment](#after-deployment)
+  - [Register the endpoint](#register-the-endpoint)
+  - [Note](#note)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Glossary
+In this process, there are 2 parties involved:
+
+**Adyen** - the payment provider which will send notifications
+**Notification module** - hosted service that accepts notifications from Adyen,
+processes and stores on a commercetools platform project.  
+
+## Requirements 
+
+#### for CTP project:
+[An `interfaceInteractions` custom type](../resources/payment-interface-interaction-types.json) should be ensured on the Notification module start
```

Comment:
Let's remove it as it is supposed to be ensured on service start-up anyway?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/45/commits/a41a45027533a281ce2f7056932145b455850c5c